rust/windows-client/src-tauri/src/client/gui.rs

```diff
@@ -265,6 +260,38 @@ pub(crate) fn run(cli: client::Cli) -> Result<(), Error> {
     Ok(())
 }
 
+async fn check_for_updates(ctlr_tx: CtlrTx, always_show_update_notification: bool) -> Result<()> {
+    let release = match client::updates::check().await {
+        Ok(x) => x,
+        Err(error) => {
+            tracing::error!(?error, "Error while checking for updates");
+            return Ok(());
+        }
+    };
```

Comment:
```suggestion
    let release = match client::updates::check().await?;
```

Reply:
Would this print a default error message? I think that should be fine

Reply:
It runs in `Tokio::spawn` that never gets joined, so if I don't log the error here, I'd have to log it up in the `spawn` call